reporters_db/data/reporters.json

```diff
@@ -442,6 +442,20 @@
             "variations": {}
         }
     ],
+    "Adm. Rec.": [
+        {
+            "cite_type": "specialty",
+            "editions": {
+                "Adm. Rec.": {
+                    "end": null,
+                    "start": null
+                }
+            },
+            "mlz_jurisdiction": [],
+            "name": "Admiralty Records.",
```

Comment:
Extra period?

Reply:
Yes, sorry. I pasted it to avoid having some typos when writing it by hand.